src/game_state/game.py

```diff
@@ -1,19 +1,112 @@
+import random
+import time
+
+import pygame
+
+from model.fruit import Fruit
+from model.snake import Direction, Snake
+from renderer.render import Color, Renderer
+
+
 class Game:
-    """Representation of a game."""
+
+    snake_speed = 15
+
+    # Window size
+    window_x = 720
+    window_y = 480
 
     def __init__(self):
-        """Initialize with some configuration maybe?
+        pygame.init()
+        pygame.display.set_caption("Snake")
+        self.game_window = pygame.display.set_mode((self.window_x, self.window_y))
+        self.fps = pygame.time.Clock()
+
+        self.snake = Snake(position=(100, 50), direction=Direction.RIGHT, size=4)
+        self.fruit = Fruit(
+            position=(
+                random.randrange(1, (self.window_x // 10)) * 10,
+                random.randrange(1, (self.window_y // 10)) * 10,
+            )
+        )
+        self.score = 0
+        self.fruit_spawn = True
+
+    def touch_boundaries(self) -> bool:
+        return (
+            self.snake.position[0] < 0
+            or self.snake.position[0] > self.window_x - 10
+            or self.snake.position[1] < 0
+            or self.snake.position[1] > self.window_y - 10
+        )
+
+    def touch_body(self) -> bool:
+        return any(self.snake.position == block for block in self.snake.body[1:])
+
+    def game_over(self):
```

Comment:
Although it is very uncommon, it is possible to win the game. This happens when all the board is occupied by the snake